src/libraries/System.Private.CoreLib/src/System/Boolean.cs

```diff
@@ -223,6 +202,9 @@ public static bool Parse(string value)
         public static bool TryParse([NotNullWhen(true)] string? value, out bool result) =>
             TryParse(value.AsSpan(), out result);
 
+        // AggressiveOptimization here gives VM a hint to bypass Tier0 and have a version where Equals is unrolled by RyuJIT
+        // so we can parse configs on start quicker
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
```suggestion
```
I do think AggressiveOptimization is appropriate here. The unrolling should be present in R2R version of this method.
`AggressiveOptimization` is start up time regression. It disables R2R and forces JIT to run with max optimizations. We do not have AggressiveOptimization in other places that call string.Equals either.

Reply:
Oops, don't know why I forgot about R2R